tests/framework/extensions/users/usermap.go

```diff
@@ -0,0 +1,21 @@
+package users
+
+import (
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	password "github.com/rancher/rancher/tests/framework/extensions/users/passwordgenerator"
+	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+)
+
+func UserConfig() (user *management.User) {
```

Comment:
Public functions need their own godoc comment explanation. Could we add an explanation?